test/scripts/e2e.sh

```diff
@@ -82,10 +95,9 @@ export TESTDIR=${TEMPDIR}
 export TESTDATADIR=${SRCROOT}/test/testdata
 export SRCROOT=${SRCROOT}
 
-./e2e_go_tests.sh
+./e2e_go_tests.sh ${GO_TEST_ARGS}
 
 rm -rf ${TEMPDIR}
-rm -rf ${PKG_ROOT}
```

Comment:
don't you need this one when 
`! ${NO_BUILD}` ?

Reply:
@tsachiherman The build_packages script now cleans up the platform root before it builds.

Reply:
yes, build_packages is ok to clean up before creating new packages.
however, it would be nice if this script would leave a clean slate after it's execution - after all, the goal of e2e.sh is to conduct the e2e test, not to perform a build. Once the test is completed, there is no point in leaving these files around.

Reply:
@tsachiherman OK, added back the removal of PKG_ROOT with conditional: ! NO_BUILD.